### ISSUE TITLE
Priest eternally falling

### ISSUE THREAD

**Issue by Flopy3987** [dalaranwow/dalaran-wow#5239]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
My priest is constantly falling so I can't even logout. I need to close the game.

**Expected behaviour**: Tell us what should happen instead
Instead of this, my character should be in Tranquillen next to the flight master.

**Steps to reproduce the problem**:

1. I used the heartstone
2. I was selling things to the innkeeper and suddenly touched the space bar accidentally and appeared there, as if I got out of the world. Then i could try to logout and reappeared in the place I was before. Went outside to get a flight but the npc disappeared so i logged out.
3. Next time I relogged I appear again in that place but moving, so i couldn't logout.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/35187306/34647526-b528a514-f363-11e7-9724-7024791efbef.png)

**Comment by TeridanDW** [comment]
Hello @Flopy3987,
Make an in-game ticket and talk to a GMs, they will assist you.
